From 8f693bc3a278a98d9e022b4c078c7f8715a73e9a Mon Sep 17 00:00:00 2001
From: Ofer Heifetz <oferh@marvell.com>
Date: Mon, 28 Dec 2015 16:59:14 +0200
Subject: [PATCH 132/538] tools: nas: fix samba share permission

smb.conf create permission was 0666 which caused samba4
to fail BD install, change create permission to have execute.

Change-Id: Ia0f8b86313ddc96ef9b490813daa6dfbd2897737
Signed-off-by: Ofer Heifetz <oferh@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/26364
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Lior Amsalem <alior@marvell.com>
---
 tools/nas/nas_init.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/nas/nas_init.sh b/tools/nas/nas_init.sh
index 3cdd8a7..e340afb 100755
--- a/tools/nas/nas_init.sh
+++ b/tools/nas/nas_init.sh
@@ -619,7 +619,7 @@ cat << EOF > $SMBCONF
 	local master = no
 	dns proxy = no
 	ldap ssl = no
-	create mask = 0666
+	create mask = 0777
 	directory mask = 0777
 	show add printer wizard = no
 	printcap name = /dev/null
-- 
1.9.1

